src/main/webapp/itempagescript.js

```diff
@@ -68,12 +128,22 @@ function getItemPageComments(comments) {
   const commentContainer = $('#comment-container');
   comments.forEach((commentDataManager) => {
     const date = new Date(commentDataManager.timestampMillis);
-    commentContainer.append(createListElement(
-        commentDataManager.comment + ' - ' +
-        '(' + date.toLocaleString() + ')'));
+    commentContainer.append($(createListElement(
+        commentDataManager.email + ': ' + commentDataManager.comment + ' - ' +
+        '(' + date.toLocaleString() + ')')));
   });
 }
 
+/**
+ * Adds Login to comment button when is not logged in
+ */
+function showLoginLink(loginUrl) {
```

Comment:
it doesn't seem like this function is called anywhere but it could be called in `loadItemPage()`